tools/docker/Dockerfile.jinja

```diff
@@ -104,6 +108,77 @@ RUN rm -rf sysbench-1.0.20 \
     iozone3_506 \
     fio
 
+#= Install applications =======================================================
+
+FROM build-base as build-applications
+
+# Install SQLite and SQLite3
+WORKDIR /root
+RUN apt-get install -y sqlite sqlite3
+RUN wget https://www.sqlite.org/2024/sqlite-amalgamation-3460100.zip \
+    && unzip sqlite-amalgamation-3460100.zip \
+    && cd sqlite-amalgamation-3460100 \
+    && gcc -g -shared -fPIC -c sqlite3.c \
+    && gcc -g -shared -fPIC -o libsqlite3.so sqlite3.o \
+    && mv ./libsqlite3.so /lib/x86_64-linux-gnu/ \
+    && mv ./sqlite3.h /usr/include/x86_64-linux-gnu/ \
+    && mv ./sqlite3ext.h /usr/include/x86_64-linux-gnu/
```

Comment:
Are you sure that these header files should be put into `/usr/include/x86_64-linux-gnu/`. Looks a bit of strange to me.

Reply:
Yes, these files are needed for LevelDB to generate the SQLite3 benchmark.